custom_components/unifi_insights/api/base.py

```
@@ -400,4 +422,4 @@ async def __aexit__(
         exc_tb: TracebackType | None,
     ) -> None:
         """Exit async context manager."""
-        await self.close()
+        await self.close()
```

Ruff (0.15.9)

[warning] No newline at end of file

Add trailing newline

(W292)